perfkitbenchmarker/linux_benchmarks/netperf_benchmark.py

```diff
@@ -240,18 +242,18 @@ def _ParseNetperfOutput(stdout, metadata, benchmark_name,
     hist_metadata.update(metadata)
     latency_samples.append(sample.Sample(
         '%s_Latency_Histogram' % benchmark_name, 0, 'us', hist_metadata))
-
-  for metric_key, metric_name in [
-      ('50th Percentile Latency Microseconds', 'p50'),
-      ('90th Percentile Latency Microseconds', 'p90'),
-      ('99th Percentile Latency Microseconds', 'p99'),
-      ('Minimum Latency Microseconds', 'min'),
-      ('Maximum Latency Microseconds', 'max'),
-      ('Stddev Latency Microseconds', 'stddev')]:
-    if metric_key in results:
-      latency_samples.append(
-          sample.Sample('%s_Latency_%s' % (benchmark_name, metric_name),
-                        float(results[metric_key]), 'us', metadata))
+  if unit != MBPS:
```

Comment:
Nope - the for loop isn't inside the `if enable_latency_histograms` block - that's why the latency samples for TCP_STREAM started showing up after your change